### PR TITLE
New Cheatsheet: Minecraft

### PR DESCRIPTION
**Pull request by kbhat95**
**What does your Instant Answer do?**
Provides PC shortcuts for the popular game Minecraft.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Zero-click info!

**What is the data source for your Instant Answer? (Provide a link if possible)**
[This](http://minecraft.gamepedia.com/Controls) is where I got my data from.

**Why did you choose this data source?**
It's the official Minecraft list of shortcuts.

**Are there any other alternative (better) data sources?**
I don't think so.

**What are some example queries that trigger this Instant Answer?**
* minecraft cheatsheet
* minecraft cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Gamers, definitely. Minecraft is one of the most popular games out there today, so it should be used by a lot of people. :)

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes; you can view it [here](https://duck.co/ideas/idea/5994/default-key-bindings-for-recent-popular-games).

**Which existing Instant Answers will this one supercede/overlap with?**
None.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot from 2015-06-30 09 51 02

## Checklist
```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    I've tested on Ubuntu 14.04, in the Chromium web browser.

```

------
IA Page: https://duck.co/ia/view/minecraft_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/minecraft.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                "val": "View list of players"
+            }
+        ],
+        "Debug": [
+            {
+                "key": "[1] to [9]", 
+                "val": "Increase/decrease graph detail"
+            },
+            {
+                "key": "[Shift] + [F3]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/minecraft.json]: We *just* released an update to the Cheat Sheet syntax. We've decided to ditch the ` + ` between consecutive key presses. Can you please remove them here?

Comment (moollaza) [inline share/goodie/cheat_sheets/minecraft.json]: Should now be: `[Shift][F3]`

**Comment by kbhat95** [comment]
@moollaza um, IMO this looks pretty weird. Is this how it's supposed to look?

screen

**Comment by javathunderman** [comment]
@kbhat95 That looks about right.

**Comment by kbhat95** [comment]
@javathunderman oh, okay. For some reason, I like the old style better.

Anyway, you probably had good reasons for the switch!

@moollaza is this ready for merge? :)

**Comment by RomainLG** [comment]
@moollaza I liked the `[shift]`+`[F3]` style better than `[shift][F3]` !

Looks less confusing. I had a double take about whether it was one key or the other or two at the same time.

**Comment by moollaza** [comment]
@kbhat95 @RomainLG I made a mistake -- you need a space between the keys still: e.g. `[Ctrl] [Shift]`

**Comment by kbhat95** [comment]
@moollaza agreed, this looks better.

screenshot from 2015-07-03 09 14 51

**Comment by moollaza** [comment]
@kbhat95 this LGTM :+1:  

Sorry about the delay!

**Comment by kbhat95** [comment]
@moollaza thanks!

**Comment by moollaza** [comment]
@kbhat95 congrats! It's live: https://duckduckgo.com/?q=minecraft+cheat+sheet&ia=cheatsheet

Sorry for the delay! It's been live for a little while...

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and @comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by kbhat95** [comment]
@moollaza thanks!

**Pull request merged**